src/lando/settings.py

```diff
@@ -198,7 +196,10 @@
     "lando.main.auth.LandoOIDCAuthenticationBackend",
 ]
 
-LINT_PATHS = tuple(f"{BASE_DIR}/{path}" for path in ("api", "dockerflow", "main", "utils", "tests"))
+LINT_PATHS = tuple(
+    f"{BASE_DIR}/{path}"
+    for path in (".", "ui", "api", "dockerflow", "main", "utils", "tests")
+)
```

Comment:
Any reason we couldn't just run `black` on all files by default? Maybe it would make more sense to maintain a list of exclude paths in the `tool.black` section of `pyproject.toml` instead of maintaining a list of include paths?

Reply:
These changes affect both `black` and `ruff` by the way.

I guess this is a remnant of the legacy implementation (we've always explicitly set the paths this way) though this is less relevant (or completely irrelevant perhaps) now since all relevant files are under `src/lando`.

In the future, we might want to add additional paths to this that are outside `src/lando`, so maybe this should just be changed to 
```suggestion
LINT_PATHS = (f"{BASE_DIR}/{path}", )
```